### ISSUE TITLE
KoboldCPP-v1.79.1.yr1-ROCm Unable to run.

### ISSUE THREAD

**Issue by DocAmakus** [YellowRoseCx/koboldcpp-rocm#91]
**Describe the Issue**
When I hit launch in the GUI it will go for a bit before CTD.

**Additional Information:**
Windows 11
CPU: AMD Ryzen 7 5800X 8-Core Processor 
32.0 GB Ram
GPU: AMD Radeon RX 7900 XT
Version: KoboldCPP-v1.79.1.yr1-ROCm
Unfortunately I do not see any logs in the folder the exe is in if you can tell me where it is saved I will be happy to provide it

Settings are all default I did not adjust anything aside from context size to 32768.

However in the command prompt when launching it does give this error before CTD.

In the command prompt window
rocBLAS error: Cannot Read C:\Users\Drama\AppData\Local\Temp\_MEI369642\/library/TensileLibrary.dat No such file or directory for GPU arch : gfx1100

![4GoyA3YniG](https://github.com/user-attachments/assets/1e6836f0-1c22-456a-8d14-201e88c173f9)

I will note it ran perfectly fine on the version right before it. KoboldCPP-v1.79.1.yr0-ROCm

**Comment by daMustermann** [comment]
You have a 7900XTX. The current release is missing the files for it. The issue tracker wasn't open when I posted it, but here is a solution: https://github.com/YellowRoseCx/koboldcpp-rocm/discussions/86

**Comment by YellowRoseCx** [comment]
> You have a 7900XTX. The current release is missing the files for it. The issue tracker wasn't open when I posted it, but here is a solution: #86

Thanks for letting me know, I'll try to get that fixed asap

**Comment by daMustermann** [comment]
Yes, the gfx1100 files are now included and it works smoothly. Thank you!

**Comment by DocAmakus** [comment]
It is indeed working again thank you.

**Comment by jeremymeyers** [comment]
@YellowRoseCx These seem to be missing in 1.98 again? There are no files with gfx1*  in C:\Users\[username]\AppData\Local\Temp\_MEI646602\library\
